### DISCUSSION TITLE
三个模式的区别仅仅是初始prompt不一样吗？

### DISCUSSION THREAD

**Discussion by Ofnoname** [viopsa233/AI-testers-Forum#52] [Q&A]
RT。有没有董哥了解

**Comment by ghost** [comment]
当然不是，创造模式是最聪明的模式

**Reply by Ofnoname** [reply]
对 这个明显可以感受到，我觉得，三模式的模型可能就不一样，是经过不同调参出来的

**Reply by ghost** [reply]
平衡模式是GPT-3.5，精确模式未知，但是和GPT-3.5一样输出不了《爱莲说》

**Reply by Ofnoname** [reply]
> 平衡模式是GPT-3.5，精确模式未知，但是和GPT-3.5一样输出不了《爱莲说》

啊 难道只有创造是gpt4吗
